builders/interoperability/xcm/send-execute-xcm.md

```diff
@@ -102,11 +102,11 @@ The `execute` function of the Polkadot XCM Pallet accepts two parameters: `messa
     ```
 
 4. Specify the `maxWeight`, which includes a value for `refTime` and `proofSize` that you will need to define:
-    - The `refTime` is the amount of computational time that can be used for execution. For this example, you can set it to `100000000000n`
-    - The `proofSize` is the amount of storage in bytes that can be used. You can set this to `0`
+    - The `refTime` is the amount of computational time that can be used for execution. For this example, you can set it to `400000000` since the `refTime` for [`WithdrawAsset`](https://github.com/moonbeam-foundation/moonbeam/blob/{{networks.moonbase.spec_version}}/pallets/moonbeam-xcm-benchmarks/src/weights/moonbeam_xcm_benchmarks_fungible.rs#L37){target=_blank} and [`DepositAsset`](https://github.com/moonbeam-foundation/moonbeam/blob/{{networks.moonbase.spec_version}}/pallets/moonbeam-xcm-benchmarks/src/weights/moonbeam_xcm_benchmarks_fungible.rs#L59){target=_blank} is set to `200000000` each
+    - The `proofSize` is the amount of storage in bytes that can be used. You can set this to `14484` since the `proofSize` for [`WithdrawAsset`](https://github.com/moonbeam-foundation/moonbeam/blob/{{networks.moonbase.spec_version}}/pallets/moonbeam-xcm-benchmarks/src/weights/moonbeam_xcm_benchmarks_fungible.rs#L37){target=_blank} and [`DepositAsset`](https://github.com/moonbeam-foundation/moonbeam/blob/{{networks.moonbase.spec_version}}/pallets/moonbeam-xcm-benchmarks/src/weights/moonbeam_xcm_benchmarks_fungible.rs#L59){target=_blank} is set to `7242` each
```

Comment:
These links have changed in: https://github.com/moonbeam-foundation/moonbeam-docs/pull/737/commits/2da7144a2a8ea32abf6e119478b855106e0806db#diff-9668549d3ea32ff3d454a3fcb3a8822badb767e96fb84850761519744bb2016f